canvas/image.md

```diff
@@ -30,21 +32,79 @@ func main() {
 	myApp := app.New()
 	w := myApp.NewWindow("Image")
 
-	image := canvas.NewImageFromResource(theme.FyneLogo())
-	// image := canvas.NewImageFromURI(uri)
-	// image := canvas.NewImageFromImage(src)
-	// image := canvas.NewImageFromReader(reader, name)
-	// image := canvas.NewImageFromFile(fileName)
-	image.FillMode = canvas.ImageFillOriginal
+	r := theme.AccountIcon()
+	image := canvas.NewImageFromResource(r)
 	w.SetContent(image)
+	w.Resize(fyne.NewSize(200, 200))
+	w.ShowAndRun()
+}
+```
+
+In our example we use `theme.AccountIcon()`, because it is already available in the Fyne library. To use your own images you can bundle them into your project, which will make them available as Fyne resources. For more information please see [Bundling resources]({% link extend/bundle.md %}).
+
+Please note that we are setting a size for the window. This is necessary, because images normally do not have a minimum size and Fyne would draw a window with zero width and height. For more details please see the section about [Fillmode](#fillmode).
+
+## Image from a URL
+
+Images can also be created from URLs with `canvas.NewImageFromURI()`. This will fetch the image during runtime from a public image URL.
```

Comment:
I find this misleading as you can't pass a URL into a URI method.

You might find it easier to use `LoadResourceFromURLString` instead.